### DISCUSSION TITLE
Install problem: php-xml

### DISCUSSION THREAD

**Discussion by V96GLF** [magicbug/Cloudlog#1411] [General]
I'm doing a new install on Raspbian Bullseye, attempting to follow the Wiki install instructions for Linux.

For PHP, the recommended install seemed to work fine:

`sudo apt-get install php7.3 php7.3-curl php7.3-mysql php7.3-mbstring php7.3-xml php7.3-openssl`

All seems well until I get to the "Update Country Files" menu option, at which point I get:

`You must install php-xml for this to work.`

I also tried apt-get install php-xml, which I think might have installed a later version of php-xml, but I get the same error. For info:

`php-xml is already the newest version (2:8.1+92+0~20220117.43+debian11~1.gbpe0d14e).`

I'm stuck now. Any suggestions?

**Comment by magicbug** [comment]
If you run `php -m` on the command line is xml listed?

**Reply by V96GLF** [reply]
No, it isn't. I guess that's a good clue ;-)

**Comment by V96GLF** [comment]
Solved.

I installed PHP 7.4 (rather than 7.3), and the problem went away.
